Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,181 @@
+
+         
+            Console.WriteLine("Please enter your weight in kg.");
+            float weightInKg = float.Parse(CheckStringForDigitsOnly());
+
+          
+            Console.WriteLine("Please enter your height in meters.");
+            float height = float.Parse(CheckStringForDigitsOnly());
+
+         
+            Console.WriteLine("Thank you for your information. You entered:");
+
+            Console.WriteLine($"\n``` {firstName} {lastName} is {age} years old, your weight is {weightInKg} kg and" +
+                              $" your height is {height} cm. ```");
+
+            
+            CalculateBmi(weightInKg, height);
+        }
+
+
+        //Make sure string has only letters
+        public static string CheckStringForLettersOnly()
+        {
+            bool stop = true;
+            string testString = "";
+
+            while (stop)
+            {
+                testString = Console.ReadLine();
+
+                if (!IsAllLetters(testString))
+                {
+                    Console.WriteLine("You entereted that wrong. Please try again.\n");
+                }
+                
+                else
+                {
+                    stop = false;
+                }
+            }
+            
+            return testString;
+        }
+
+        //check each character in string to see if it has any numbers
+        public static bool IsAllLetters(string testForLetters)
+        {
+            foreach (char c in testForLetters)
+            {
+               
+                if (!Char.IsLetter(c))
+                {
+                    return false;
+                }
+            }
+         
+            return true;
+        }
+
+        //Make sure string has only numbers
+        public static string CheckStringForDigitsOnly()
+        {
+            bool stop = true;
+            string testDigits = "";
+
+            while (stop)
+            {
+                testDigits = Console.ReadLine();
+
+                if (!IsAllDigits(testDigits))
+                {
+                    Console.WriteLine("You entereted that wrong. Please try again.\n");
+                }
+                else
+                {
+                    stop = false;
+                }
+            }
+
+            return testDigits;
+        }
+
+        //Make sure string is only numbers also allowed to have '.'
+        public static bool IsAllDigits(string testForNumbers)
+        {
+            foreach (char c in testForNumbers)
+            {
+                if (!Char.IsDigit(c) && c != '.')
+                {
+                    return false;
+                }
+            }
+            return true;
+        }
+
+
+        //For Checks
+        public static int ConvertToInt(string message)
+        {
+           // Console.WriteLine("Type something and it will be returned as an int.");
+            int toAnInt = int.Parse(message);
+           // Console.WriteLine($"The converted entry is: {toAnInt}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4852921)